raiden_contracts/tests/test_channel_withdraw.py

```diff
@@ -0,0 +1,436 @@
+import pytest
+from eth_tester.exceptions import TransactionFailed
+from web3.exceptions import ValidationError
+from raiden_contracts.utils.events import check_withdraw
+from .fixtures.config import empty_address, fake_bytes
+from raiden_contracts.utils.config import (
+    E_CHANNEL_WITHDRAW,
+    CHANNEL_STATE_NONEXISTENT_OR_SETTLED,
+    CHANNEL_STATE_OPEN,
+    CHANNEL_STATE_CLOSED,
+    SETTLE_TIMEOUT_MIN
+)
+from .utils import MAX_UINT256
```

Comment:
Maybe not for this PR, but we tend to not use relative imports at all. You use them quite a bit. Can you at some point change them to full path imports? e.g. for here: `from raiden_contracts.test.utils import MAX_UINT256` ?

Reply:
In case of files from the same directory it seems a bit too much to restrict to full paths. Why is this?

Reply:
It's I would say 50% a style choice and 50% being future-proof for if you move the file itself out of the directory the imports would still work.

Reply:
Same for me - we changed folder structure a couple of times and relative paths seemed better because of this.
But I didn't know that full paths were the chosen style for Raiden, so I'll change.

Reply:
Thank you. Does not need to be in this PR, it's already quite bit. Just remember it for later.